package.json

```diff
@@ -43,6 +43,6 @@
     ]
   },
   "devDependencies": {
-    "cypress": "^9.5.4"
+    "cypress": "^9.7.0"
```

Comment:
What's the reason for bumping the cypress version? If you ran `npm install`, this might have happened automatically. Please revert this (and the package-lock.json) unless there's a good reason for it